po/es.po

```diff
@@ -6589,11 +6594,11 @@ msgstr ""
 
 #: src/ownership/moved-strings-rust.md:10
 msgid "The heap data from `s1` is reused for `s2`."
-msgstr "La pila (heap) de datos de `s1` es rehusado para `s2`."
+msgstr "Los datos del _heap_ de `s1` se reutilizan en `s2`."
 
 #: src/ownership/moved-strings-rust.md:11
 msgid "When `s1` goes out of scope, nothing happens (it has been moved from)."
-msgstr "Cuando `s1` sale del scope, no sucede nada (se movió de)."
+msgstr "Cuando `s1` sale del ámbito, no ocurre nada (se ha movido)."
```

Comment:
You can also use the same time form: "ha sido movida" (the problem of this form I'm suggesting is that it's not as common, and it also forces us to set the gender of movido/movida, which depends on the gender of what is referring to)